CMakeLists.txt

```diff
@@ -289,9 +289,6 @@ if(CONFIG_OVERRIDE_FRAME_POINTER_DEFAULT)
   endif()
 endif()
 
-separate_arguments(CONFIG_COMPILER_OPT_AS_LIST UNIX_COMMAND ${CONFIG_COMPILER_OPT})
```

Comment:
probably should be named something else, but the key here is CONFIG_COMPILER_OPT which is being parsed and set later.

Reply:
Ok, going to drop that change from my PR.